draft-ietf-httpbis-safe-method-w-body.xml

```diff
@@ -908,13 +948,280 @@ See stored query at "/contacts/stored-queries/42".
+<polygon class="arrowhead" points="344,400 332,394.4 332,405.6" fill="black" transform="rotate(0,336,400)"/>
+<polygon class="arrowhead" points="344,208 332,202.4 332,213.6" fill="black" transform="rotate(0,336,208)"/>
+<polygon class="arrowhead" points="344,80 332,74.4 332,85.6" fill="black" transform="rotate(0,336,80)"/>
+<polygon class="arrowhead" points="24,464 12,458.4 12,469.6" fill="black" transform="rotate(180,16,464)"/>
+<polygon class="arrowhead" points="24,272 12,266.4 12,277.6" fill="black" transform="rotate(180,16,272)"/>
+<polygon class="arrowhead" points="24,144 12,138.4 12,149.6" fill="black" transform="rotate(180,16,144)"/>
+<g class="text">
+<text x="28" y="36">Client</text>
+<text x="316" y="36">Resource</text>
+<text x="92" y="68">QUERY with content</text>
+<text x="308" y="116">200 OK</text>
+<text x="276" y="132">Validator: foo</text>
+<text x="92" y="180">QUERY with content</text>
+<text x="108" y="196">(conditional on 'foo')</text>
+<text x="268" y="244">304 Not Modified</text>
+<text x="276" y="260">Validator: foo</text>
+<text x="348" y="324">State Change</text>
+<text x="92" y="372">QUERY with content</text>
+<text x="108" y="388">(conditional on 'foo')</text>
+<text x="308" y="436">200 OK</text>
+<text x="276" y="452">Validator: bar</text>
+</g>
+</svg>
+<?ENDINC flow-query.svg ?>
+</artwork>
+</artset></figure>
+      <figure title="Data Flow with GET to Equivalent Resource">
+        <artset>
+          <artwork type="ascii-art" name="flow-query-get" align="center"><![CDATA[
+Client                             Resource
+|                                         |
+| QUERY with content                      |           Equivalent Resource
++---------------------------------------->|              (generates /xyz)
+|                                         +-----------------------------o
+|                                         |                             |
+|                                  200 OK |                             |
+|                          Validator: foo |                             |
+|                          Location: /xyz |                             |
+|<----------------------------------------+                             |
+|                                                                       |
+| GET                                                                   |
+| (conditional on 'foo')                                                |
++---------------------------------------------------------------------->|
+|                                                                       |
+|                                                      304 Not Modified |
+|                                                        Validator: foo |
+|<----------------------------------------------------------------------+
+|                                                                       |
+|                                                                +--------------+
```

Comment:
I make this line out at 81 columns, which is why things end up looking a little squished.